hardhat.config.ts

```diff
@@ -193,4 +194,19 @@ const config: HardhatUserConfig = {
     },
 };
 
+/**
+ * Ignore some contracts from compilation. Credits to:
+ * https://kennysliding.medium.com/how-to-ignore-solidity-files-in-hardhat-compilation-6162963f8c84
+ */
+subtask(TASK_COMPILE_SOLIDITY_GET_SOURCE_PATHS).setAction(async (_, __, runSuper) => {
```

Comment:
How much time do we win doing this ? It's intresting since we don't use this contract anymore. Is it temporary => if we want to delete this later or keep it as archive ?

Reply:
We want to delete it as soon as possible, in another PR to separate works :)

Reply:
Does the compilation succeed if we don't ignore these files ?
It's very useful to know how to ignore contracts (thanks) but I'm thinking is it worth adding new code for contracts that we will remove anyway ?

Reply:
Compilation fails without that. PR would be huge if cleaning is done in this PR. What is your opinion?

Reply:
I agree with you, cleaning can be done separately.
We can keep it since the compilation fails.